definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-only-for-prod.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "LiveFrom": "25/06/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "IssueAOSOffline_Respondent_fromAosAwaiting",
```

Comment:
Don't forget to apply this to all events starting with "IssueAOSOffline_"

Reply:
When you say apply this, do you mean adding full "CRUD" settings to these options for beta behind a feature toggle?

Reply:
@jacmurphy

Reply:
Updated @jvictorcf @yarpo